UnrealEngine.gitignore

```diff
@@ -43,6 +50,22 @@
 *.sdf
 *.VC.db
 *.VC.opendb
+.idea/
+.kdev4/
+.vscode/
+*.code-workspace
```

Comment:
I think these other rules are reasonable to include if you can break it out into it's own section with a header describing where they come from